### DISCUSSION TITLE
System.IO.FileNotFoundException  Message=Could not load file or assembly 'Microsoft.AspNetCore.Razor.Runtime, Version=2.0.2.0

### DISCUSSION THREAD

**Discussion by asadsahi** [Dotnet-Boxed/Templates#188] [Q&A]
﻿**Which project template is the issue about?**
AspNetCore + Angular 
...

**Which version of the project template is the issue about?**
3.5.0
...

**Which version of Visual Studio or the dotnet SDK were you using?**
Windows 10
VS 2017, latest
with these sdks installed on machine
1.0.4 [C:\Program Files\dotnet\sdk]
1.1.8 [C:\Program Files\dotnet\sdk]
2.1.101 [C:\Program Files\dotnet\sdk]
2.1.300-preview1-008174 [C:\Program Files\dotnet\sdk]
...

Getting this error when running the project

![image](https://user-images.githubusercontent.com/3470892/37434749-9c0cc1f6-27d8-11e8-9086-612cc264e9af.png)

[Here](https://github.com/asadsahi/AspNetCoreBoilerplate) is OOTB Repo

**Comment by RehanSaeed** [comment]
Looks like you have preview1 installed. Add a global.json to pin the version to use 2.0.

**Comment by asadsahi** [comment]
@RehanSaeed thanks, just realised, I wanted to raise it against other asp.net core boilerplate repo :)

That was the exact issue BTW.

Thanks again.
